pkg/aggregator/aggregator.go

```diff
@@ -209,19 +209,43 @@ func renameDefinition(s *spec.Swagger, old, new string) {
 	delete(s.Definitions, old)
 }
 
-// MergeSpecsOrFailOnConflict is differ from MergeSpecs as it fails if there is
+// MergeSpecsIgnorePathConflict is the same as MergeSpecs except it will ignore any path conflicts.
```

Comment:
ignoring by what? by keeping the entry for dest or source?

Reply:
fixed.